### PR TITLE
add the other Federalist repos to documentation

### PR DESCRIPTION
**Pull request by wslack**
This commit adds and describes all Federalist repos

### CODE REVIEW

File: pages/how-federalist-works.md [hunk 1]
```diff
@@ -6,21 +6,27 @@ title: How Federalist Works
...
+- **[federalist-docker-build](https://github.com/18F/federalist-docker-build)** Pulling from docker-ruby-ubuntu, this container image adds the dependencies and scripts for a Garden Linux container to build out a Federalist site.
+- **[docker-ruby-ubuntu](https://github.com/18F/docker-ruby-ubuntu)** Container image specification to run in a Garden Linux container.
+
+### Templates and Documentation
+
+- **[federalist-docs](https://github.com/18F/federalist-docs)** This documentation website. It also contains content from the depreciated [federalist-content-guide](https://github.com/18F/federalist-content-guide) repo.
+- **[federalist-modern-team-template](https://github.com/18F/federalist-modern-team-template)** A simple site to showcase the work of an organization with a few different page types.
+- **[federalist-landing-page-template](https://github.com/18F/federalist-landing-page-template)** A single page meant to host information about a new team or report
```
Comment (brandocalrissian) [inline pages/how-federalist-works.md]: totally nitpicking, but the other bullets end with a period while this one doesn't.

Totally minor minor minor minor can't say how minor...

File: pages/how-federalist-works.md [hunk 2]
```diff
@@ -6,21 +6,27 @@ title: How Federalist Works
...
-- **[federalist-docs](https://github.com/18F/federalist-docs)** This documentation website
+- **[Federalist](https://github.com/18F/federalist)** This is the core application for Federalist and contains the frontend interface as well as the code that interacts with the GitHub API and sets up new Federalist sites, including a Github webhook that triggers messages into the build queue through the app. It has two cloud.gov service instances for session storage and persistant data storage of users, sites, and logs.
+- **[federalist-builder](https://github.com/18F/federalist-builder)** This application launches build tasks for Federalist in a Linux Garden container based on messages from a queue and contains scheduling logic.
+- **[federalist-registry](https://github.com/18F/federalist-registry)** This application hosts the Federalist Docker Build image for the federalist-builder application to reference. It uses an [AWS storage driver](https://docs.docker.com/registry/storage-drivers/s3/).
```
Comment (jmhooper) [inline pages/how-federalist-works.md]: We use "Federalist Docker Build" here and "docker-ruby-ubuntu" on the next line. We probably want to keep the way we use the names of the repos consistent.

File: pages/how-federalist-works.md [hunk 3]
```diff
@@ -6,21 +6,27 @@ title: How Federalist Works
...
+### Functional Repositories
 
-- **[federalist-builder](https://github.com/18F/federalist-builder)** This application is used to launch build tasks for Federalist in an AWS ECS Docker container based on messages from an AWS SQS queue
-- **[federalist-docker-build](https://github.com/18F/federalist-docker-build)** A Docker image that runs Jekyll to build a site
-- **[docker-ruby-ubuntu](https://github.com/18F/docker-ruby-ubuntu)** A base image for Docker
-- **[federalist-modern-team-template](https://github.com/18F/federalist-modern-team-template)** This is a Jekyll site. It is a simple site to showcase the work of an organization with a few different page types.
-- **[federalist-docs](https://github.com/18F/federalist-docs)** This documentation website
+- **[Federalist](https://github.com/18F/federalist)** This is the core application for Federalist and contains the frontend interface as well as the code that interacts with the GitHub API and sets up new Federalist sites, including a Github webhook that triggers messages into the build queue through the app. It has two cloud.gov service instances for session storage and persistant data storage of users, sites, and logs.
```
Comment (wslack) [inline pages/how-federalist-works.md]: committing edits for these in a sec

File: pages/how-federalist-works.md (hunk 1 above)
Comment (wslack) [inline pages/how-federalist-works.md]: fixed in new commit

**Comment by wslack** [comment]
Made edits! Added a note that the "docker" names of our repos don't mean we actually use Docker.

**Comment by brandocalrissian** [comment]
looks good to me! merging...

**Pull request merged**